tests/commands/install_uninstall_test.py

```diff
@@ -617,9 +617,9 @@ def test_install_allow_mising_config(tempdir_factory):
         ret, output = _get_commit_output(tempdir_factory)
         assert ret == 0
         expected = (
-            '`.pre-commit-config.yaml` config file not found. '
+            '`{}` config file not found. '
```

Comment:
this actually makes the test less readable:

https://testing.googleblog.com/2014/07/testing-on-toilet-dont-put-logic-in.html

also this is unrelated, please rebase this out.